src/index.js

```diff
@@ -471,12 +474,131 @@ class XRState {
 }
 const xrState = GlobalContext.xrState = new XRState();
 
-if (nativeBindings.nativeVr) {
+if (nativeBindings.oculusVr) {
+  nativeBindings.nativeVr.requestPresent = function (layers) {
```

Comment:
I think this is slightly confusing to hijack `nativeBindings.nativeVr` (which was previously OpenVR) to bring up the Oculus device. Maybe it would be better to have both drivers separate rather than decorating each other's functions?

Reply:
OpenVR and Oculus headsets are now decoupled.